src/pages/concept-list-page/concept-list-page-pure.tsx

```diff
@@ -36,9 +37,11 @@ export const ConceptListPagePure = ({ history, concepts, publisher, catalogId }:
     <div className="row mb-2">
       <ConceptTitle title={getTitle(publisher)} />
     </div>
-    <div className="mb-2">
-      <NewConceptButton parentOnClick={() => createNewConceptAndNavigate({ history, catalogId })} />
-    </div>
+    <Can I="create" of={{ publisher: catalogId, __type: 'Concept' }}>
```

Comment:
men det  er jo ikke engelsk? "can i create of" mener ingenting? er "for" eler "in" ikke tillatt? vel du kan jo også lese params rett fra props:

men det  er jo ikke engelsk? "can i create of" mener ingenting? er "for" eler "in" ikke tillatt? vel du kan jo også lese params rett fra props:

kan du ikke bruke type lenger som "a"?
```
<Can I="create" a="concept" of={{ publisher: catalogId}}>
```
ellers, hvis du ønsker, kan du dekorere litt (hoc) den Can med ekstra handy parametrer:
```
 <Can I="create" a="concept" forPubliser={catalogId}>
```

Reply:
Har forsøkt å si <Can I="create" a="concept" of={{ publisher: catalogId}}>, men det fungerer ikke.

Kan du vise meg hvordan jeg kan dekorere, evt. par programming på det som gjenstår?